tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -40,7 +40,13 @@ class Camera(Base):
     def launch(self):
         Base.launch(self)
         self.wait_for_capture_ready()
-        self.wait_for_element_not_displayed(*self._loading_screen_locator)
+        from marionette_driver.errors import (NoSuchElementException,
+                                              StaleElementException)
+        try:
+            Wait(self.marionette, ignored_exceptions=StaleElementException).until(
+                lambda m: not m.find_element(*self._loading_screen_locator).is_displayed())
+        except NoSuchElementException:
+            pass
```

Comment:
I'' be doing the same thing here then, regarding your comment about: Use expected.element_not_displayed from http://marionette-client.readthedocs.org/en/latest/reference.html#marionette_driver.expected.element_not_displayed